Update docu with info about image build process (#108)
packit-service/deployment: README.md

- git branch -f stable commit-hash
- git push [-u upstream] stable

#### Image build process

> Images builds are triggered by new commits via dockerhub. ([dockerhub autobuild docs](https://docs.docker.com/docker-hub/builds/))

Reviewer: ```suggestion
Image builds are triggered by new commits on Docker Hub. ([Autobuild docs](https://docs.docker.com/docker-hub/builds/))
```

> In packit-service we are using custom build hook to be able inject ENV variable

Reviewer: ```suggestion
In packit-service we are using a custom build hook to be able inject the ENV variables
```

Reviewer 2: ```suggestion
In packit-service we use a custom build hook to be able to inject ENV variables
```

> provided by build process ([docs](https://docs.docker.com/docker-hub/builds/advanced/))

Reviewer: ```suggestion
provided by the build process ([docs](https://docs.docker.com/docker-hub/builds/advanced/))
```

Reviewer 2: ```suggestion
provided by the build process. ([docs](https://docs.docker.com/docker-hub/builds/advanced/))
```
